packages/webapp/src/pages/Team/Billing/Show.tsx

```diff
@@ -0,0 +1,134 @@
+import { IconExternalLink } from '@tabler/icons-react';
+import { useMemo } from 'react';
+import { Helmet } from 'react-helmet';
+import { Link } from 'react-router-dom';
+
+import { ErrorPageComponent } from '../../../components/ErrorComponent';
+import { LeftNavBarItems } from '../../../components/LeftNavBar';
+import { Skeleton } from '../../../components/ui/Skeleton';
+import { useEnvironment } from '../../../hooks/useEnvironment';
+import { useApiGetPlans } from '../../../hooks/usePlan';
+import DashboardLayout from '../../../layout/DashboardLayout';
+import { useStore } from '../../../store';
+import { cn } from '../../../utils/utils';
+
+import type { PlanDefinition } from '@nangohq/types';
+
+interface PlanDefinitionList {
+    plan: PlanDefinition;
+    active: boolean;
+    below?: boolean;
+    above?: boolean;
+}
+
+export const TeamBilling: React.FC = () => {
+    const env = useStore((state) => state.env);
+
+    const { error, plan: currentPlan, loading } = useEnvironment(env);
+    const { data: plansList } = useApiGetPlans(env);
+
+    const plans = useMemo<PlanDefinitionList[]>(() => {
+        if (!currentPlan || !plansList) {
+            return [];
+        }
+
+        // No self downgrade or old plan
+        if (currentPlan.name === 'scale' || currentPlan.name === 'enterprise' || currentPlan.name === 'starter' || currentPlan.name === 'internal') {
+            return [{ plan: plansList.data.find((p) => p.code === currentPlan.name)!, active: true }];
+        }
+
+        const list: PlanDefinitionList[] = [];
+        let passedActive = false;
```

Comment:
I had to read the passedActive logic twice to understand what it actually means? Would it make more sense to call below/above something like canUpgrade/Downgrade?

Reply:
yeah right, will rename that!